mirror.go

```diff
@@ -40,6 +40,8 @@ func _RecursiveMirror(source, dest reflect.Value, bestEffort bool) error {
 	sourceKind := source.Kind()
 
 	switch sourceKind {
+	case reflect.Invalid:
+		return nil
```

Comment:
If it's not convertible at all, we should return an error. Otherwise it will be a silent error which is dangerous if the data is expected to be exist.

Reply:
Hi, i am not sure but

empty interface{} is convertible for example to nil pointer or to any default value of field

as i understand empty interface reflected as  Invalid

Reply:
Ok i updated code - so SmartMirror will silently pass, but Mirror will complain

Reply:
Sorry a bit busy right now but thanks for making a fix for this